sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -135,24 +181,31 @@ mod tests {
     #[test]
     fn test_create_account_full() {
         let def_hash = Hash::default();
-        let account =
-            create_account_with_data(42, vec![(0u64, &def_hash); MAX_ENTRIES].into_iter());
+        let def_fees = FeeCalculator::default();
+        let account = create_account_with_data(
+            42,
+            vec![IterItem(0u64, &def_hash, &def_fees); MAX_ENTRIES].into_iter(),
+        );
         let recent_blockhashes = RecentBlockhashes::from_account(&account).unwrap();
         assert_eq!(recent_blockhashes.len(), MAX_ENTRIES);
     }
 
     #[test]
     fn test_create_account_truncate() {
         let def_hash = Hash::default();
-        let account =
-            create_account_with_data(42, vec![(0u64, &def_hash); MAX_ENTRIES + 1].into_iter());
+        let def_fees = FeeCalculator::default();
+        let account = create_account_with_data(
+            42,
+            vec![IterItem(0u64, &def_hash, &def_fees); MAX_ENTRIES + 1].into_iter(),
+        );
         let recent_blockhashes = RecentBlockhashes::from_account(&account).unwrap();
         assert_eq!(recent_blockhashes.len(), MAX_ENTRIES);
     }
 
     #[test]
     fn test_create_account_unsorted() {
-        let mut unsorted_recent_blockhashes: Vec<_> = (0..MAX_ENTRIES)
```

Comment:
@ryoqun would you mind giving this a once-over to ensure I preserved the intended logic?

Reply:
Yeah, I've checked and this pr is preserving the intended logic.

Reply:
Thank you!